packages/fx-core/src/plugins/resource/function/resources/errors.ts

```diff
@@ -243,6 +243,12 @@ export class PublishCredentialError extends FunctionPluginError {
   }
 }
 
+export class FindAppError extends FunctionPluginError {
```

Comment:
What about `FunctionAppNotFoundError`?

Reply:
In post-provision, the plugin will get the function resource and prepare to update it.
This error means finding function failed.